database/sql/src/main/scala/cromwell/database/slick/SlickDatabase.scala

```diff
@@ -100,6 +100,8 @@ class SlickDatabase(override val originalDatabaseConfig: Config) extends SqlData
     actionThreadPool, database.executor.executionContext.reportFailure
   )
 
+  protected[this] lazy val insertBatchSize = databaseConfig.as[Option[Int]]("db.insertBatchSize").getOrElse(2000)
```

Comment:
the camelCase goes against our typical convention but an astute reader will notice that the other config settings in the db block are camelCase as well